test/harmonytest.js

```diff
@@ -18524,9 +18524,9 @@ var harmonyTestFixture = {
         },
 
         'func(...a, b)': {
-            index: 9,
+            index: 10,
             lineNumber: 1,
-            column: 10,
+            column: 11,
```

Comment:
This is the space after the comma, which feels like the wrong column to point to.